projects/demo/src/modules/components/editor/examples/8/index.html

```diff
@@ -0,0 +1,11 @@
+<tui-editor
+    class="editor"
+    [formControl]="control"
+    [tools]="builtInTools"
+></tui-editor>
+
+<h4>HTML:</h4>
+<tui-editor-socket [content]="control.value || ''"></tui-editor-socket>
```

Comment:
https://taiga-ui--pr2931-splincode-fix-editor-f2e4ekk9.web.app/components/editor#anchors

https://user-images.githubusercontent.com/35179038/198984860-0199bd68-157e-4372-b378-d80be379ca45.mov

Reply:
Problem of firebase-app only ?

Or the production https://taiga-ui.dev will have the same problem ?

Reply:
fixed